tools/ci-build/scripts/check-style-and-lints

```diff
@@ -8,6 +8,6 @@ set -eux
 cd workspace/smithy-rs
 sdk-lints check --all
 sdk-lints fix --all
-rustfmt --check --edition 2018 $(find -name '*.rs' -print | grep -v /target/)
+rustfmt --check --edition 2018 "$(find . -name '*.rs' -print | grep -v /target/)"
```

Comment:
hmm...I think one of these works on gnu find and one doesn't IIRC?

Reply:
I tried the command out on a Mac, and it seems to work fine there. Regardless, these scripts get run in the Docker build image, which has the GNU coreutils.